metadata/static/knative/logo.svg

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this the official logo from here: https://drive.google.com/drive/u/0/folders/1huGB3psIe7vwbJxL3GWX-nmtetL6SFW4 ?

We should not use the OSS version.